src/style.scss

```diff
@@ -354,13 +407,23 @@ div.tooltip {
   hyphens: auto;
 }
 
+/**
+This class adds a fading effect and hides the element
+ */
 .invisibleClass {
-  transition: opacity 1s linear;
+  transition: opacity 0.1s linear;
   opacity: 0;
   height: 0;
   overflow: hidden;
 }
 
+/**
+This class hides the element totally;
+ */
+.invisibleClass2 {
```

Comment:
Remove this and use the `.hidden` class instead: http://getbootstrap.com/css/#helper-classes-show-hide